src/main/kotlin/com/itiviti/DotnetPlugin.kt

```diff
@@ -66,14 +65,13 @@ class DotnetPlugin: Plugin<Project> {
                 exec.standardOutput = outputStream
             }
             val versionString = outputStream.toString()
-            val targetFramework = if (versionString.startsWith("3.1")) {
-                "netcoreapp3.1"
-            } else {
-                if (!versionString.startsWith("5.0")) {
-                    project.logger.info("Default to use target framework to .Net Sdk 5.0, please make sure it is installed")
-                }
-                "net5.0"
+
+            val targetFramework = when {
+                versionString.startsWith("3.1") -> "netcoreapp3.1"
+                versionString.startsWith("6.0") -> "net6.0"
+                else -> "net5.0"
```

Comment:
by default I would go for net6.0 LTS, but since net6.0 is still under preview let's keep net5.0 for now and change to net6.0 when released

Reply:
thinking again, since it is using dotnet --version to get the targetFramework and it is expected to get the version
should we do like
```
            val targetFramework = when {
                versionString.startsWith("3.1") -> "netcoreapp3.1"
                versionString.startsWith("5.0") -> "net5.0"
                versionString.startsWith("6.0") -> "net6.0"
                else -> throw new GradleException("Please make sure .Net SDK is installed or set the dotnet executable path correctly")
            }
```

Reply:
Yeah, lemme clean that up really quick and i'll resubmit :) Also thanks for the quick response! ^^